modules/aggregations/src/yamlRestTest/resources/rest-api-spec/test/aggregations/time_series.yml

```diff
@@ -343,3 +343,109 @@ setup:
 
   - match: { aggregations.ts.buckets.0.key: { "key": "10" } }
   - match: { aggregations.ts.buckets.0.doc_count: 1 }
+  - match: { aggregations.ts.buckets.1.key: { "key": "11" } }
```

Comment:
I think it is not an extra bucket...the test before was just testing one of the buckets instead of two and I added both just to show that the result is the same (2 buckets).

Reply:
Ok, I see. Makes sense.